protocol/v2/blockchain/beacon/network.go

```diff
@@ -4,18 +4,37 @@ import (
 	"time"
 
 	"github.com/attestantio/go-eth2-client/spec/phase0"
-
 	spectypes "github.com/bloxapp/ssv-spec/types"
 )
 
 // Network is a beacon chain network.
 type Network struct {
 	spectypes.BeaconNetwork
+	LocalTestNet bool
 }
 
 // NewNetwork creates a new beacon chain network.
 func NewNetwork(network spectypes.BeaconNetwork) Network {
-	return Network{network}
+	return Network{
+		BeaconNetwork: network,
+		LocalTestNet:  false,
+	}
+}
+
+// NewNetworkWithLocalTestNet creates a new beacon chain network.
```

Comment:
Could you please adjust the comment? Also, I know I suggested `NewNetworkWithLocalTestNet` but now I'm not sure it's a good name. Maybe `NewLocalTestNetwork`?